deploy/resources/v0.8.0/02-clustertasks/buildah-task.yaml

```diff
@@ -0,0 +1,6 @@
+# auto generated by script/update-tasks.sh
+# DO NOT EDIT: use the script instead
+# source: https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.7/buildah/buildah-task.yaml
+#
+---
+
```

Comment:
but it's empty though :angel:

Reply:
good find, the same with s2i task. I know why !.. `cat <<-EOF` does not fail :(